snewpdag/plugins/DiffPointing.py

```diff
@@ -156,6 +155,26 @@ def reevaluate(self, data):
     return data
 
   def alert(self, data):
+    ###################fake dts: alert nu_times being sent from the server right now are of the order of seconds
```

Comment:
not sure I understand what this section is supposed to do (probably I'm missing something)